code/style.css

```diff
@@ -1,4 +1,359 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffffff;
+  color:grey;
 }
+
+/*header layout*/
+.header {
+  background-image: url("https://images.unsplash.com/photo-1497105943914-b90d55145023?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1270&q=80");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: auto;
+  }
+  
+  .flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
+  
+  .logo {
+    width: 20%;
+    height: auto;
+    padding-left: 20px;
+    padding-top: 20px;
+  }
+  
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    font-family: Verdana, sans-serif;
+    font-weight: bolder;
+    font-size: 22px;
+  }
+
+  .nav-item {
+    margin: 10px;
+    padding: 10px;
+  }
+  
+/*big news and main layout*/
+.main {
+  background-color: #C5E3D5;
+  padding: 10px 0 10px 0;
+  font-family: verdana, sans-serif;
+  }
+
+  .big-news {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: center;  
+    background-color: #E2F3EB; 
+  }
+  
+  #img-big {
+    object-fit: scale-down;
+    width: 100%;
+    height: auto;
+  }
+
+  #big-news-h1 {
+    font-family: 'Yusei Magic', sans-serif;
+  }
+
+  #big-news-p {
+    padding-left: 30px;
+    font-size: 18px;
+  }
+
+/*links effects*/
+  a:link {
+    text-decoration: none;
+    color: #835F80;
+  }
+
+  a:hover {
+    text-decoration: none;
+    color: darkgrey;
+  }
+
+  a h1:link {
+    text-decoration: none;
+    color: grey;
+  }
+
+  a h1:hover {
+    text-decoration: none;
+    color: darkgrey;
+  }
+
+  a h3:link {
+    text-decoration: none;
+    color: grey;
+  }
+
+  a h3:hover {
+    text-decoration: none;
+    color: darkgrey;
+  }
+
+  a #img-big:hover {
+    opacity: 0.5;
+  }
+
+  a #first:hover{
+    opacity: 0.5;
+  }
+
+  a #second:hover{
+    opacity: 0.5;
+  }
+
+  a #third:hover{
+    opacity: 0.5;
+  }
+
+  a #fourth:hover{
+    opacity: 0.5;
+  }
+
+/*other news (section) layout*/
+  .other-news h3 {
+    font-family: 'Yusei Magic', sans-serif;
+  }
+
+  .other-news-item {
+    text-align: center;
+    background-color: #E2F3EB;
+  }
+
+  #first {
+    object-fit: scale-down;
+    width: 100%;
+  }
+
+  #second {
+    object-fit: scale-down;
+    width: 100%;
+  }
+
+  #third {
+    object-fit: scale-down;
+    width: 100%;
+  }
+
+  #fourth {
+    object-fit: scale-down;
+    width: 100%;
+  }
```

Comment:
Same here. If you were to style many elements in the same way though, you could do it with a comma:
```
.element-1, .element-2 {
  color: red;
}
```